features/review_reminder.feature

```diff
@@ -0,0 +1,25 @@
+Feature: Review reminders
+  In order to allow publishers to maintain content
+  An editor
+  Should be able to set and edit a review date for a document
+
+  Background:
+  Given I am an editor
+
+  Scenario: Creating a new draft publication with a review date
+    When I start drafting a new publication "Standard Beard Lengths"
+    And I add a review date of "2033-1-1" and the email address "test@gmail.com" on the edit page
+    Then I should see the review date of "1 January 2033" on the edition summary page
```

Comment:
It shows that the review date has been persisted the database correctly though right? I guess this could be handled via a controller spec though.

Reply:
Yeah, it does show that. I guess this feature test covers the CRUD functionality of that, rather than the email being sent out. (Which is covered elsewhere in other tests.)

So yeah, deffo nothing to change. It still shows you can set a date on an unpublished doc.